### PR TITLE
[MIG] project_task_material_stock: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
If CIs go well

**Review by chienandalu — COMMENTED** [review]
Am I doing something wrong?

peek 21-09-2018 14-34

**Comment by cubells** [comment]
@chienandalu your are right. I fix this problem soon.

**Comment by cubells** [comment]
@chienandalu can you retest this pr? thanks.

**Review by chienandalu — APPROVED** [review]
Great! Now works fine :slightly_smiling_face:

File: project_task_material_stock/README.rst [hunk 1]
```diff
@@ -46,32 +67,43 @@ Bug Tracker
...
+Authors
+~~~~~~~
+
+* Tecnativa
+
 Contributors
-------------
+~~~~~~~~~~~~
+
+* Tecnativa <https://www.tecantiva.com>:
```
Comment (pedrobaeza) [inline project_task_material_stock/README.rst:86]: Incorrect URL and you should put correctly the link format in RST:
```
* `Tecnativa <https://www.tecnativa.com>`_:
```

File: project_task_material_stock/models/project_task.py [hunk 2]
```diff
@@ -49,6 +48,10 @@ def _compute_stock_state(self):
         compute='_compute_stock_move',
         string='Stock Moves',
     )
+    analytic_account_id = fields.Many2one(
+        comodel_name='account.analytic.account',
+        string='Analytic Account',
```
Comment (pedrobaeza) [inline project_task_material_stock/models/project_task.py:53]: Put a help attribute explaining for what it is.

File: project_task_material_stock/models/project_task.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015 Antiun Ingeniería S.L. - Sergio Teruel
```
Comment (pedrobaeza) [inline project_task_material_stock/models/project_task.py]: Change copyright to new company

File: project_task_material_stock/models/project.py [hunk 4]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015 Antiun Ingeniería S.L. - Sergio Teruel
```
Comment (pedrobaeza) [inline project_task_material_stock/models/project.py]: Change copyright to new company

File: project_task_material_stock/__manifest__.py [hunk 5]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015 Antiun Ingeniería S.L. - Sergio Teruel
```
Comment (pedrobaeza) [inline project_task_material_stock/__manifest__.py]: Change copyright to new company

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@cubells please finish this PR

File: project_task_material_stock/tests/test_task_material.py [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        })
+        cls.product1_uom = cls.env.ref('product.product_uom_unit')
+        cls.product2_uom = cls.env.ref('product.product_uom_kgm')
+        cls.product1 = cls.env['product.product'].create({
+            'name': 'Product example #1',
+            'uom_id': cls.product1_uom.id,
+            'uom_po_id': cls.product1_uom.id,
+            'company_id': cls.company.id,
+        })
+        cls.product2 = cls.env['product.product'].create({
```
Comment (nikul-serpentcs) [inline project_task_material_stock/tests/test_task_material.py]: @cubells IMO here use like global variable

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Comment by cubells** [comment]
I think it's all done @pedrobaeza

**Review by nikul-serpentcs — APPROVED** [review]
Thanks, @cubells 

Code Review LGTM

**Review by pedrobaeza — APPROVED** [review]

**Comment by jcarlosmontoya** [comment]
Hello guys, I just test this module, but when click Consume Materials Stock moves stay always in state assigned, doing something wrong? best regards

**Comment by jcarlosmontoya** [comment]
@chienandalu Does it work fine this module now?

**Comment by jcarlosmontoya** [comment]
Note: Stock moves are in assigned status and move.quantity_done always zero, super method _action_done() return empty recordset

**Comment by cubells** [comment]
@jcarlosmontoya I'll review later.

**Comment by jcarlosmontoya** [comment]
@cubells Thank you

**Comment by jcarlosmontoya** [comment]
@cubells I just do this: If you set quantity_done = requested qty (task line product_uom_qty) before to call action_done method works fine
Can i do  a PR?

**Comment by jcarlosmontoya** [comment]
This example shows how to create a move_line_ids for each stock.move before to call _action_done method
https://github.com/odoo/odoo/blob/12.0/addons/stock/models/stock_inventory.py#L430

**Pull request merged**